### DISCUSSION TITLE
TSC meeting and slack technical issues

### DISCUSSION THREAD

**Discussion by Xerkus** [laminas/technical-steering-committee#54] [General]
@laminas/technical-steering-committee so, today is our scheduled TSC meeting but slack seems to be inoperable.
Hopefully service will be fully restored but if not what would be our plan?

**Comment by Xerkus** [comment]
There is currently no items on the agenda. First meeting after long holidays. Should we maybe try a different format  - here, in discussions?

**Reply by geerteltink** [reply]
I'm fine with `pre-discussing` things here. And final details and votes during meetings.

**Reply by Xerkus** [reply]
Not a chat, but when everyone is in attendance we can make it work as a pretty robust replacement I think.
I mean, in this format the discussion would allow for a more informative messages that give just enough pause to think that extra second but without the need for a more fleshed out response we would normally have in the issues or on forums.

Also full power of GH issues platoform, code samples, cross-links and indefinite history.

**Reply by Xerkus** [reply]
The threading seems to be good too. Definitely visible, unlike slack.

**Reply by weierophinney** [reply]
But does it use websockets and push changes? If not... it's no synchronous enough for meeting-style discussions.

**Reply by Xerkus** [reply]
It should update immediate without reload. Issues were like that for a while. Unless may be connectivity is bad

**Reply by weierophinney** [reply]
Okay, it does reload... but the interface jumps around A TON when it does, making accessibility and following painful.

Also, I have to tab 7 times to do the default action from wthin a reply or write dialog. ick.

**Reply by weierophinney** [reply]
Oh, FFS. I also GET AN EMAIL FOR EVERY REPLY. Even when I have the tab open.

**Reply by samsonasik** [reply]
"Unsubscribe" button should make no send to email for new reply?

**Reply by Xerkus** [reply]
Yep. It definitely changes the dynamic of the conversation.

This could work if meeting format is changed to asynchronous during specific hours on a specific day and then have like a synchronous vote phase as a conclusion.

**Reply by Xerkus** [reply]
Also longer reply threads get folded
